lib/fog/vsphere/models/compute/rules.rb

```diff
@@ -0,0 +1,28 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Rules < Fog::Collection
+        autoload :Rule, File.expand_path('../rule', __FILE__)
+        
+        model Fog::Compute::Vsphere::Rule
+        attr_accessor :datacenter
+        attr_accessor :cluster
```

Comment:
Yeap. It does. The benefits is that you can use the `requires` method to tell a method needs that variable to no be nil.